### DISCUSSION TITLE
Direct bind between switch and bulb and still have mqtt messages

### DISCUSSION THREAD

**Discussion by piketres** [Koenkk/zigbee2mqtt#6271] [Devices]
Is it possible to directly bind a switch (Aqara Opple) to a bulb (AwoX) to have a fast on/off action, and still be able to send the 2-click action to mqtt so that I can manage the blinds with it?

I´ve managed to do the direct bind and be able to control the bulb but no messages are sent to mqtt. I´ve also managed to bind the switch to the bulb and also bind it to the coordinator. Doing so, I can only use the on/off function of the bulb (ok for me), single click will turn it on or off and also the action will be sent to mqtt. However, double click won´t do anything. It wont change the bulbs brightness and it won´t be sent to mqtt.

Any ideas?

**Comment by Koenkk** [comment]
Try putting the bulb in a group and binding the remote to the group

**Reply by piketres** [reply]
I tried it yesterday just after commenting here with no success. The switch binds and I can control both on/off and brightness of the bulb but only button_1_single is sent to mqtt. I suppose it has something to do with the "command" mode I need to put the switch in. 

I don´t know yet how reporting works in zigbee, or the difference between "command" and "event" mode. I don´t know if all the switches work the same way. 

I suppose that for the message to get sent to mqtt it needs to be seen by the coordinator? If that is the case then I suppose that the bulb needs to forward those events? I´m trying to understand how the inner networking works to understand where the "issue" comes from. 

Thanks Koenkk for the suggestion in any case!
